systemvm/debian/opt/cloud/bin/configure.py

```diff
@@ -1070,8 +1070,8 @@ def main(argv):
     config.address().process()
 
     databag_map = OrderedDict([("guest_network",     {"process_iptables": True,  "executor": []}),
-                               ("vm_password",       {"process_iptables": False, "executor": [CsPassword("vmpassword", config)]}),
-                               ("vm_metadata",       {"process_iptables": False, "executor": [CsVmMetadata('vmdata', config)]}),
+                               ("vm_password",       {"process_iptables": True, "executor": [CsPassword("vmpassword", config)]}),
+                               ("vm_metadata",       {"process_iptables": True, "executor": [CsVmMetadata('vmdata', config)]}),
```

Comment:
@weizhouapache can you explain how does this fix the issue? (I've no idea how the key is used, however ignore my comment if the PR is still WIP)